### ISSUE TITLE
Term "intellectual property" used in PeerTube 2.3

### ISSUE THREAD

**Issue by mdimjasevic** [Chocobozzz/PeerTube#3010]
**What happened?**

I was reading the [PeerTube 2.3 release notes](https://joinpeertube.org/en_US/news#release-2-3-0) and noticed that in the section on video reporting PeerTube offers an option to report due to a "intellectual property" violation.

 
**What do you expect to happen instead?**

Given that PeerTube uses the term "free software" and not some other more popular terms to talk about software freedom, i.e., given that it pays attention to terminology, I would expect it not to use the term "intellectual property". Instead, given that a video is not a physical object, I would expect some other term to be used. A detailed explanation why the term is problematic can be found at https://www.gnu.org/philosophy/not-ipr.en.html.

**Steps to reproduce:**

1. Visit https://joinpeertube.org/en_US/news#release-2-3-0

2. Scroll down to the section "New features on content moderation".

3. Notice the term in a screenshot.

**Comment by rigelk** [comment]
What alternative are you suggesting?

**Comment by Dragoncraft89** [comment]
I think a big part of report reasons would be copyright infringement.

But copyright infringement is not "intellectual property".

Intellectual property contains a lot of laws, like patent law or trademark law, which do not fit here. Maybe trademark law could be a reason if the video contains unlawful uses of brand icons, but it's hard to search for other applications here.

On the other hand it does not cover every law that is applicable. For example, Germany prohibits "Nazi symbolism" in non-educational media, which doesn't fit that category at all

It should be renamed to something like "Law violation" if it is desired to group every unlawful activity. Otherwise "Copyright infringement" should be enough for most use cases.

**Comment by test2a** [comment]
@rigelk hey. i have a question about reporting videos also. The announcement says report video option would be visible to logged in users only. why have we restricted only logged in users to report a problem video? many instances do not allow registrations for whatever reason. I just checked reddit and youtube and they both have a report button for logged out users and it asks them to log in to report. What if the said peertube instance does not allow registrations? shouldnt we be able to intimate the administrator about a potential problem even then?
